plugins/msg-broker/mcollective/lib/openshift/mcollective_application_container_proxy.rb

```diff
@@ -1815,7 +1815,8 @@ def move_gear_post(gear, destination_container, state_map)
       if gear.group_instance.platform.downcase == "windows"
         log_debug "DEBUG: Restoring ownership and user ACLs for Windows gear '#{gear.uuid}'"
         rsync_keyfile = Rails.configuration.auth[:rsync_keyfile]
-        log_debug `eval \`ssh-agent\`; ssh-add #{rsync_keyfile}; ssh -o StrictHostKeyChecking=no -A root@#{destination_container.get_ip_address} "/cygdrive/c/openshift/bin/oo-cmd.exe oo-admin-restore-acls --uuid:#{gear.uuid}"; exit_code=$?; ssh-agent -k;exit $exit_code`
+        destination_address = Rails.configuration.msg_broker[:node_address_by_fqdn] ? destination_container.id : destination_container.get_ip_address
```

Comment:
@sosiouxme : I realized as soon as I started testing in a devenv that calling it FQDN was probably wrong, but I didn't realize that there were cases were the ID was not a hostname at all, so thanks for raising that point.  I'll think about an appropriate name change.